thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/common/ThirdEyeConfiguration.java

```diff
@@ -174,4 +168,11 @@ public void setDataSources(String dataSources) {
     this.dataSources = dataSources;
   }
 
+  public Map<String, Map<String, String>> getAlerterConfiguration() {
```

Comment:
Maybe change the return type to Map<String, Object>, would add more flexibility.

Reply:
I am not very confident of making it an Object. These are platform metadata. I don't foresee any benefits by making it an object. In addition, making it an object will require us to call toString each time we extract the properties.

Reply:
just in case the configuration might be a list or something else later. such as 

>   alerter_configuration: prop: [value, value]

thirdeye usually uses a MapUtils library to get properties out of Map<String, Object> configs.

not a blocker.